tests/acceptance/stepDefinitions/privateLinksContext.js

```diff
@@ -0,0 +1,26 @@
+const { client } = require('nightwatch-api')
+const { When } = require('cucumber')
+const webdav = require('../helpers/webdavHelper')
+
+When('the user copies the private link of the file/folder {string} using the webUI', function (resource) {
+  return client.page.FilesPageElement
+    .filesList()
+    .clickRow(resource)
+    .copyPrivateLink()
+})
+
+When('the user navigates to the copied private link using the webUI', function () {
+  return client.getClipBoardContent(function (url) {
+    // If the redirect happens immediately we receive an error
+    // Cannot read property 'parentNode' of null
+    setTimeout(function () {
+      client.url(url)
+    }, 500)
```

Comment:
is it a matter of time or is it just a matter of deferring the call to the event queue ? in the latter case a value of "0" would be enough and would not slow down the tests

Reply:
It works fine with 0 as well. Changed